examples/helm/index.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+import * as helm from "@pulumi/kubernetes/helm";
+
+const nginx = new helm.v2.Chart("simple-nginx", {
+  // Represents chart `stable/nginx-lego@v0.3.1`.
+  repo: "stable", chart: "nginx-lego", version: "0.3.1",
+  values: {
+    // Override for the Chart's `values.yml` file. Zero out resource requests so it can be scheduled
+    // on our (wimpy) CI cluster.
+    nginx: {resources: null},
+    default: {resources: null},
+    lego: {resources: null},
+  },
+  transformations: [
+    // Don't expose any services publicly, i.e., turn all services into ClusterIP instead of
+    // LoadBalancer. (This cuts time to run CI tests significantly.)
+    helm.transform.ifGvkEqual("v1", "Service", (svc: any) => {
```

Comment:
Let's get rid of `helm.transform`, and just write the `if` inline here.  It'll be *a lot* clearer what's going on here and how to do other similar things.